src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -400,6 +400,30 @@ await _context.Entry(loadedMethodology.ScheduledWithRelease)
                 });
         }
 
+        public async Task<Either<ActionResult, List<MethodologyVersionViewModel>>> ListMethodologyVersionsForApproval(Guid userId)
+        {
+            var publicationIdsForApprover = _context
```

Comment:
Same question for `ListReleasesForApproval`

Reply:
Hey Mark,

I'd say probably not, because the filtering of Methodologies is tailored specifically to the user.  If the user does not have permission to approve any Methodologies, no Methodologies would be returned for them.

Same goes for the Release equivalent.

I suppose it could be argued that there is some potential for accidental abuse of this as the `userId` being used with this method doesn't *necessarily* have to be the Id of the user performing the action if someone was accidentally misusing this code, so it could be argued that the userId shouldn't be a parameter of this method but should instead be grabbed from UserService.GetUserId() *within* this method.  Then, no accidental misuse of this method!

Or I could just add a usage comment to the method!  What do you think?

Reply:
I've internalised the fetching of the User's Id into this method.